### PR TITLE
Fix SSL translation

### PR DESCRIPTION
**Pull request by pierallard**
Fix remark from Crowdin

Exact acronym is "SSL", not "Ssl".

https://crowdin.com/translate/akeneo/291/en-hu#7107

### CODE REVIEW

File: src/Pim/Bundle/UserBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -68,7 +68,7 @@ Business Units: Business Units
 Email synchronization settings: Email synchronization settings
 Host: Host
 Port: Port
-Ssl: Ssl
+Ssl: SSL
```
Comment (grena) [inline src/Pim/Bundle/UserBundle/Resources/translations/messages.en.yml]: :walking:

**Comment by grena** [comment]
G T M

File: src/Pim/Bundle/UserBundle/Resources/translations/messages.en.yml (hunk 1 above)
Comment (pierallard) [inline src/Pim/Bundle/UserBundle/Resources/translations/messages.en.yml]: :dash:

**Comment by loicguillois** [comment]
👍

**Pull request merged**